### DISCUSSION TITLE
Is it impossible to create an own Rest API like "antmedia.io/rest"

### DISCUSSION THREAD

**Discussion by williehao** [ant-media/Ant-Media-Server#5864] [FAQs]
Is it impossible to create an own Rest API like "antmedia.io/rest" with EE or CE version 
![image](https://github.com/ant-media/Ant-Media-Server/assets/15116422/c2d37c4f-0b75-4b0d-98d6-1c5dffff1d45)

**Comment by lastpeony** [comment]
Hi @williehao 
Lets continue our discussion at
https://github.com/orgs/ant-media/discussions/5868
